content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Using OpenTelemetry Instrumentation Libraries with Datadog SDKs
+further_reading:
+    - link: 'tracing/guide/instrument_custom_method'
+      text: 'Instrument a custom method to get deep visibility into your business logic'
+    - link: 'tracing/connect_logs_and_traces'
+      text: 'Connect your Logs and Traces together'
+    - link: 'tracing/visualization/'
+      text: 'Explore your services, resources, and traces'
+    - link: 'https://www.datadoghq.com/blog/opentelemetry-instrumentation/'
+      text: 'Learn More about Datadog and the OpenTelemetry initiative'
+algolia:
+  tags: ['otel instrumentation']
+---
+
+[Instrumentation][1] is the process of adding code to your application to capture and report observability data.
+[Automatic instrumentation][2] is a way to instrument applications and libraries without directly modifying their source code.
+Both OpenTelemetry and Datadog provide automatic instrumentations as part of their SDKs.
+
+Datadog SDKs support adding [instrumentation libraries][3] from OpenTelemetry to their existing automatic instrumentations.
+This provides observability for libraries not originally covered by Datadog SDKs without needing to change SDKs.
+
+## Prerequisites
+
+Before adding OpenTelemetry instrumentation libraries, set the `DD_TRACE_OTEL_ENABLED` environment variable to `true`.
+
+<div class="alert alert-warning">
+When replacing an existing Datadog instrumentation with its OpenTelemetry equivalent, remember to disable the
+Datadog instrumentation to avoid duplicate spans in the trace.
+</div>
+
+## Language support
+
+| Language | Minimum version required |
+|----------|--------------------------|
+| Java     | v1.35.0                  |
+| Python   | v2.10.0                  |
+| Ruby     | v2.1.0                   |
+| Go       | -                        |
+| Node.js  | -                        |
+| PHP      | -                        |
+| .NET     | -                        |
+
+{{< tabs >}}
+
+{{% tab "Java" %}}
+
+## Compatibility requirements
+
+The Datadog SDK for Java supports library instrumentations written using OpenTelemetry's [instrumentation API][4] and `javaagent` [extension API][5].
+
+Each instrumentation must be packaged as an OpenTelemetry [extension][6] in its own jar.
+
+OpenTelemetry provide an [example extension project][7] that provides a custom [instrumentation for Servlet 3 classes][8].
```

Comment:
provides

Reply:
There are also 2 uses of 'provides' in this sentence which is not great - I'll fix that

Reply:
c0487e49dc